UI/HTML/adminpage.html

```diff
@@ -0,0 +1,342 @@
+                  </form>
+               </div>
+          </div>
+          <div id ="orders">
+              <span class = "ordered">3 pizza   50$ </span>
+              <div class="radio">
+                  <form >
+                    <span> accept<input type="radio" name = "mark">  </span>
+                    <span> decline<input type="radio"  name = "mark"> </span>
+                    <span>complete <input type="checkbox"> </span>
+                    <span><input class="finish"  type = "submit" value = "finish"/> </span>
+                  </form>
+               </div>
+            </div>
+          <div id ="orders">
+              <span class = "ordered">3 plain chichen ,1 passon juice.  50$ </span>
+               <div class="radio">
+                 <form >
+                   <span> accept<input type="radio" name = "mark">  </span>
+                   <span> decline<input type="radio"  name = "mark"> </span>
+                   <span>complete <input type="checkbox"> </span>
+                   <span><input class="finish"  type = "submit" value = "finish"/> </span>
+                 </form>
+              </div>
+            </div>
+
+      </div>
+     
+      <div id="div2" class="tabcontent">
+          <div id ="orders">
+            <span class = "ordered">3 plain chicken ,1 passon juice. 50$ </span>
+            <strike class = "status">2 hrs ago. cancelled</strike>
+           </div>
+         <div id ="orders">
+            <span class = "ordered">3 chips and chicken  ,1 passon juice.  50$ </span>
+             <strong class="status">7hrs ago.  pending..</strong>
+        </div>
+        <div id ="orders">
+            <span class = "ordered">1 plain rice ,1 cocktail juice.  50$ </span>
+            <strong class="status">yesterday. completed</strong>
+         </div>
+        <div id ="orders">
+            <span class = "ordered">2 plain chichen ,1 passon juice.  50$ </span>
+             <strong class="status"> 3 days ago. completed</strong>
+        </div>
+
+        <div id ="orders">
+            <span class = "ordered">3 plain chichen ,1 passon juice.  50$ </span>
+            <strong class="status">5 daysago.  completed</strong>
+        </div>
+        <div id ="orders">
+            <span class = "ordered">3 pizza   50$ </span>
+              <strong class="status">5 das ago.  completed</strong>
+          </div>
+        <div id ="orders">
+            <span class = "ordered">3 plain chichen ,1 passon juice.  50$ </span>
+             <strong class="status">2 weeks ago. completed</strong>
+          </div>
+    </div>
+
+
+
+
+      <div id="div3" class="tabcontent">
+          <table class = "orderTable">
+              <tr>
+                  <td> 
+                    <p class = "imagetitle">chicken pizza :20$ </p>
+                    <img class = "image" src= "../IMAGES/chickenPizza.jpg">
+                    <form>
+                      <dv>
```

Comment:
This appears as <dv>, Did you mean to type <div>? Same occurs on more lines in this file.

Reply:
oh its an error . thx  @66hakeem